Plutarch/Core.hs

```diff
@@ -411,6 +341,31 @@ type PSOP edsl =
   , IsPType edsl PPType
   )
 
+class
+  ( SOP.AllZip2 (SOP.LiftedCoercible SOP.I (Term edsl)) tss pss
+  , SOP.AllZip2 (SOP.LiftedCoercible (Term edsl) SOP.I) pss tss
+  ) =>
+  IsPCodeOf edsl pss tss
```

Comment:
Could we have `| edsl pss -> tss, edsl tss -> pss`?

Reply:
Indeed we cannot, which is unfortunate. It will not allow the 'catch-all' instance of the constraint newtype pattern.

Reply:
https://github.com/Plutonomicon/plutarch-core/pull/18#pullrequestreview-1111511619

Reply:
It actually may, if we use a type family to compute `pss` from `tss`.